src/main/java/com/extendedclip/deluxemenus/hooks/ExecutableBlocksHook.java

```diff
@@ -0,0 +1,36 @@
+package com.extendedclip.deluxemenus.hooks;
+
+import com.extendedclip.deluxemenus.cache.SimpleCache;
+import com.ssomar.score.api.ExecutableBlocksAPI;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+public class ExecutableBlocksHook implements ItemHook, SimpleCache {
+
+  private final Map<String, ItemStack> cache = new ConcurrentHashMap<>();
+
+  @SuppressWarnings("deprecation")
```

Comment:
This is indeed not that nice. After looking at a decompiled version of the plugin, it seems that it does indeed have an API and the wiki suggests using the JAR directly. No remote repository is provided.

I guess, for now we'll support the deprecated version until the developer decides to completely remove it or they provide an online version of the dependency..